cmd/server.go

```diff
@@ -120,7 +94,7 @@ func runServer(cmd *cobra.Command, args []string) error {
 
 	// Load security rules based on configuration
 	var rules []*models.SpotterRule
-	rulesPaths := viper.GetStringSlice("server.rules-path")
+	rulesPaths := config.RulesPath
```

Comment:
We can improve this later, as when we run it a admission controller we wouldn't to load rules from outside the cluster.

Reply:
Make sense